src/assets/YAMLAsset.js

```diff
@@ -12,8 +13,19 @@ class YAMLAsset extends Asset {
   }
 
   generate() {
+    let code = `module.exports = ${JSON.stringify(this.ast, false, 2)};`;
+
+    if (this.options.minify) {
+      let minified = minify(code);
+      if (minified.error) {
+        throw minified.error;
+      }
+
+      code = minified.code;
+    }
```

Comment:
This looks the same as the code in JSONAsset. Maybe we could make a util function to JSON stringify and optionally minify and then use it in both places?

Reply:
Actually, I'll rewrite this without JSON.stringify, because YAML can contain dates.

Reply:
Please check the new commit